core/src/main/java/org/apache/iceberg/SystemProperties.java

```diff
@@ -41,6 +41,9 @@ private SystemProperties() {}
 
   public static final int IO_MANIFEST_CACHE_MAX_FILEIO_DEFAULT = 8;
 
+  public static final String DELETE_POS_FILES_THREAD_POOL_SIZE =
```

Comment:
The name of the variable seems to be specific to position deletes while the rest of the names are not. Is that deliberate?